arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -140,6 +143,9 @@
         "responses": {
           "200": {
             "description": "The request was successful; the request was well-formed and received properly."
+          },
+          "204": {

```

Comment:
Correct, we don't want an error thrown when deleting a resource that doesn't exist.